pages/search/index.vue

```diff
@@ -291,7 +297,21 @@
         this.rerouteSearch({ qf: this.qfForSelectedFacets, reusability: this.reusability, page: '1' });
       },
       selectView (view) {
+        if (process.browser) {
+          sessionStorage.view = view;
+          localStorage.view = view;
+        }
+        this.view = view;
         this.view = view;
+      },
+      selectedView: function () {
+        if (process.browser) {
+          if (this.$route.query.view) {
+            sessionStorage.view = this.$route.query.view;
+          }
+          return sessionStorage.view || localStorage.view || 'grid';
+        }
+        return 'grid';
```

Comment:
Should this return `this.view`?

Reply:
I think that could lead to an infinite loop, but I've made it aware of any setting in the URL params so as to prevent the observed issue of always rendering 'grid' server side.